includes/src/add_producto.php

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="/G3_SW/includes/views/estilo.css" />
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Login</title>
+<script>
+function validarFormulario() {
```

Comment:
Se agradece que no dejéis código que no funcione ya que complica el proceso revisión